HardwareDefinitions/avnet_sk_rev2_files_for_SDK_installer/avnet_aesms_mt3620_rev2.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Can you please move this to avnet_mt3620_sk_rev2?

Reply:
Done